scopes/dependencies/dependency-resolver/dependency-resolver.main.runtime.ts

```diff
@@ -934,6 +936,44 @@ export class DependencyResolverMain {
     return manifest;
   }
 
+  getOutdatedPkgs({
```

Comment:
add return value type

Reply:
I would rename it a bit. it's not just gettingOutdatedPkgs.
It get outdated packages of configured packages.
I would expect from a function that called `getOutdatedPkgs` to get a list of packages. not a list of configs.
(I do think we should have a function in the main runtime, that is called `getOutdatedPkgs` and does exactly this)

Reply:
add a jsdocs comment for this function